lib/OpenQA/Worker/Jobs.pm

```diff
@@ -344,13 +344,6 @@ sub _stop_job {
     # process of killing the backend process and checksums uploads and
     # checksums again can take a long while, so the webui needs to know
 
-    if ($aborted eq "scheduler_abort") {
-        log_debug('stop_job called by the scheduler. do not send logs');
-        _kill_worker($worker);
-        _reset_state;
```

Comment:
Check the _reset_state here... IIRC there was some mumbo jumbo happening when it was a shared worker...